src/app/common/components/PipelineSummary.tsx

```diff
@@ -35,10 +35,9 @@ export const getPipelineSummaryTitle = (status: IVMStatus): string => {
     return 'Complete';
   }
   if (status.started && !status.completed) {
-    let title: string;
-    title = status.error?.phase ? `Error - ` : '';
+    let title = status.error ? 'Error' : 'Running';
```

Comment:
@gildub was it intentional to remove the part of the condition that checks for a truthy `error?.phase` property?

The in-progress migrations in the mock data have bare error objects:
```ts
    error: {
      phase: '',
      reasons: [''],
    },
```

I'm not sure if that's true of the real data, but at least in mock mode we're getting "Error -" in front of non-error statuses because it's just checking if there is an `error` object at all. We should check whether there can be a bare error object like this in real data, and if so put back the `?.phase` part of the condition. If not we should remove those bare errors from our mock data.

Reply:
@mturley, you're right, the change is because there are no error struct unless an error occurred.
I've fixed the mock data and added a clarification in the description above.